src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/ComTypes/ITypeLib2.cs

```diff
@@ -25,9 +25,9 @@ public interface ITypeLib2 : ITypeLib
         [PreserveSig]
         new void ReleaseTLibAttr(IntPtr pTLibAttr);
         void GetCustData(ref Guid guid, out object pVarVal);
+        void GetLibStatistics(IntPtr pcUniqueNames, out int pcchUniqueNames);
         [LCIDConversion(1)]
         void GetDocumentation2(int index, out string pbstrHelpString, out int pdwHelpStringContext, out string pbstrHelpStringDll);
```

Comment:
As mentioned in the issue, `GetDocumentation2` is missing the `lcid` parameter:

https://learn.microsoft.com/en-us/windows/win32/api/oaidl/nf-oaidl-itypelib2-getdocumentation2

Reply:
It isn't missing it though, that is the point of the `LCIDConversion` attribute. The marshaller will insert that at the position indicated by the attribute during the unmanaged call.